---
name: Bug Report 🐞
about: Did you find a bug?
title: ''
labels: bug, triage
assignees: ''

---

<!--

  Hi there 👋 Thank you for discovering and submitting an issue.

  Before you submit this; let's make sure of a few things.
  Please make sure the following boxes are ticked if they are correct.
  If not, please try and fulfill these first.

-->

- [ ] I have searched the [issues] of this repo and believe that this is not a duplicate.
- [ ] I have turn-off any extension on my browser like adblock that may interfere with the app behaviour.
- [ ] If an exception occurs when executing a command, I executed it again in debug mode (`DEBUF = True` in settings).

### Describe the bug 🐛

> A clear and precise description of what the bug is, please be descriptive! Thanks 🙌

### To Reproduce 🔂

> Steps to reproduce the behavior:
>
> 1. Go to '...'
> 2. Click on '....'
> 3. Scroll down to '....'
> 4. See error

### Expected behavior 🚀

> A clear and concise description of what you expected to happen.

### Configuration ⚙️

> A clear and concise description of your configuration.
>
> Python 3.11.0
> PostgreSQL Server 14.5
> Django version 4.1.2
> ...

### Screenshots 📸

> If applicable, add screenshots to help explain your problem.

### Diagnosis attempts 🩺

> `curl "https://some.domain.name" --verbose`

### Additional context 🌍

> Add any other context about the problem here.

<!-- links -->

[issues]: https://github.com/sdurioanalytics/sdurioanalytics/issues
